src/block/accordion/index.php

```diff
@@ -18,3 +18,35 @@ function stackable_load_accordion_frontend_script() {
 	}
 	add_action( 'stackable/accordion/enqueue_scripts', 'stackable_load_accordion_frontend_script' );
 }
+
+if ( ! function_exists( 'stackable_load_accordion_frontend_polyfill_script' ) ) {
+	require_once( ABSPATH . 'wp-admin/includes/dashboard.php' );
+	/**
+	 * Adds polyfill for summary/details element that are * used in accordion blocks.
+	 *
+	 * TODO: confirm that this works on older browsers
+	 */
+	function stackable_load_accordion_frontend_polyfill_script() {
+
+		$browser_version = wp_check_browser_version();
```

Comment:
Let's not use `wp_check_browser_version`, this uses a lot more resources than necessary and also pings a remote URL.

Instead, just check `$_SERVER['HTTP_USER_AGENT']`. Be sure to check `empty( $_SERVER['HTTP_USER_AGENT'] )`